src/devneya/err.clj

```diff
@@ -5,32 +5,26 @@
   (System/exit status))
 
 (defn parse-err 
-  "Parses Throwable exception and returns status and message"
+  "Parse Throwable exception, return error message"
   [e]
-  (let  [data (get-in (Throwable->map e) [:via 0 :data])]
-    {:err-status (:status data) :err-body (:body data)}
-    )
-  )
+  (let  [data (get-in (Throwable->map e) [:via 0])] 
+    (str "Type:     "(:type data) "\nMessage:  " (:message data))))
 
 (defn catch-error
   "Main func for catching errors"
   [e]
-  (let [{:keys [err-status err-body]} (parse-err e)]
-  (exit err-status err-body))
-  )
+  (let [msg (parse-err e)]
+  (exit 1 msg)))
```

Comment:
I still think, that this function is very dangerous. We should be more careful with exits from the program and move them out of the error handling and exit only from interface itself.

Reply:
I commented on the error handling above, using the monadic error handling method this problem would be fixed.